$NetBSD: patch-ab,v 1.24 2016/06/19 05:55:57 markd Exp $

* First two chunks: automake-1.13 compatability.

Latest rrdtool doesnt have separate librrd_th

--- configure.in.orig	2011-08-15 08:34:28.000000000 +0000
+++ configure.in
@@ -56,7 +56,7 @@ dnl>
 dnl> generate the config header
 dnl>
 umask 002
-AM_CONFIG_HEADER(config.h)
+AC_CONFIG_HEADERS(config.h)
 
 HAS_ERROR=
 HAS_WARNING=
@@ -324,7 +324,7 @@ dnl>
 echo
 echo "Checking C compiler program and features..."
 echo
-AM_PROG_CC_STDC
+AC_PROG_CC
 if test ".${ac_cv_prog_cc_stdc}" = ".no"; then
     echo
     echo "*******************************************************************"
@@ -395,6 +395,19 @@ case "${target}:${CC}" in
         echo "* This appears to be ${DEFINEOS} (release not checked)"
       ;;
 
+    *-*-netbsd*:* )
+	DEFINEOS="NETBSD"
+        echo "* This appears to be ${DEFINEOS}"
+      ;;
+    *-*-openbsd*:* )
+	DEFINEOS="OPENBSD"
+        echo "* This appears to be ${DEFINEOS}"
+      ;;
+    *-*-dragonfly*:* )
+	DEFINEOS="DRAGONFLY"
+        echo "* This appears to be ${DEFINEOS}"
+      ;;
+
     * )
         echo "* ${target} is an unknown/unsupported OS"
         echo "*"
@@ -554,8 +567,8 @@ if test ".${PCAPRING_DIR}" != .; then
 else
 if test ".${PCAP_ROOT}" != .; then
   if test -d $PCAP_ROOT &&
-     test -r $PCAP_ROOT/libpcap.a &&
-     test -r $PCAP_ROOT/pcap.h; then
+     test -r $PCAP_ROOT/lib/libpcap.a -o -r $PCAP_ROOT/lib/libpcap.dylib &&
+     test -r $PCAP_ROOT/include/pcap.h; then
        PCAP_ROOT=`cd ${PCAP_ROOT} && pwd`
        CORELIBS="${CORELIBS} -L$PCAP_ROOT -lpcap"
        INCS="${INCS} -I$PCAP_ROOT"
@@ -876,19 +889,19 @@ else
    fi
 fi
 
-RRD_LIB="-L${RRD_HOME}/lib -lrrd_th"
+RRD_LIB="-L${RRD_HOME}/lib -lrrd"
 
-if test -f "$RRD_HOME/lib/librrd_th.so"; then
+if test -f "$RRD_HOME/lib/librrd.so"; then
    AC_MSG_RESULT(checking for rrdtool... yes)
 else
-  if test -f "$RRD_HOME/lib/librrd_th.dylib"; then # OSX
+  if test -f "$RRD_HOME/lib/librrd.dylib"; then # OSX
      AC_MSG_RESULT(checking for rrdtool... yes)
   else
-     if test -f "$RRD_HOME/lib/librrd_th.a"; then
+     if test -f "$RRD_HOME/lib/librrd.a"; then
        AC_MSG_RESULT(checking for rrdtool... yes)
      else
-       AC_CHECK_LIB([rrd_th], [main])
-       if test ".${ac_cv_lib_rrd_th_main}" != ".yes"; then
+       AC_CHECK_LIB([rrd], [main])
+       if test ".${ac_cv_lib_rrd_main}" != ".yes"; then
          AC_MSG_ERROR(Unable to find RRD at $RRD_HOME: please use --with-rrd-home=DIR);
 	 AC_MSG_ERROR(RRD source can be downloaded from http://www.rrdtool.org/);
        else
@@ -1721,8 +1734,6 @@ if test -f "GeoLiteCity.dat"; then
 else
    if test -f "3rd_party/GeoLiteCity.dat.gz"; then
      cp 3rd_party/GeoLiteCity.dat.gz .
-   else
-     wget http://www.maxmind.com/download/geoip/database/GeoLiteCity.dat.gz
    fi
 
    gunzip GeoLiteCity.dat.gz
@@ -1735,8 +1746,6 @@ if test -f "GeoIPASNum.dat"; then
 else
    if test -f "3rd_party/GeoIPASNum.dat.gz"; then
      cp 3rd_party/GeoIPASNum.dat.gz .
-   else
-     wget http://www.maxmind.com/download/geoip/database/asnum/GeoIPASNum.dat.gz
    fi
 
    gunzip GeoIPASNum.dat.gz
